stanford-earth/matson
# Minor fixes to sharrring.

# READY FOR REVIEW

# Summary
- Fixes the sharing issues with the share counters
- Hides the linkedin counter as that no longer works.

# Needed By (Date)
- End of sprint

# Urgency
- Low

# Steps to Test

1. Check out this branch
2. Clear all caches
3. Navigate to a news article 
4. Share a facebook post
5. See counter for facebook increase
6. Sharing for twitter wont work as it requires the earth.stanford.edu domain

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
